tests/unit/test_business_logic.py

```diff
@@ -183,6 +183,38 @@ def test_provider_request_release_files(mock_old_api):
     )
 
 
+def test_provider_get_requests_for_workspace(bll):
+    user = factories.create_user("test", ["workspace"])
+    other_user = factories.create_user("other", ["workspace"])
+    factories.create_release_request("workspace", user, id="r1")
+    factories.create_release_request("workspace", other_user, id="r2")
+
+    assert [r.id for r in bll.get_requests_for_workspace("workspace", user)] == [
+        "r1",
+        "r2",
+    ]
```

Comment:
This test would pass even if `get_requests_for_workspace` returned all requests.  I'd suggest creating a request that doesn't belong to this workspace, and ensuring that it doesn't get returned

Reply:
yep, done